### PR TITLE
refact: refact debug level logs; bump ununtu image to 20.04; switch to non-root ubuntu container

### PR DESCRIPTION
**Pull request by vutkin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,32 +23,37 @@ ENV GOOS            linux
 
 RUN go build -v -ldflags "-X main.Version=${VERSION} -s -w"
 
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 LABEL authors="Seth Miller,Yannig Perré"
 LABEL maintainer="Yannig Perré <yannig.perre@gmail.com>"
 
 ENV VERSION ${VERSION:-0.1.0}
+ENV TZ=Europe/Amsterdam \
```
Comment (Yannig) [inline Dockerfile:31]: Can you remove this env definition?

**Review by Yannig — COMMENTED** [review]
I have only one concern about the TZ env.

But otherwise it looks very good! Thanks a lot!

File: Dockerfile (hunk 1 above)
Comment (vutkin) [reply Dockerfile]: Fixed

**Comment by Yannig** [comment]
Nice. Thanks a lot!

**Pull request merged**